cmd/nodeup/main.go

```diff
@@ -50,6 +52,9 @@ func main() {
 	target := "direct"
 	flag.StringVar(&target, "target", target, "Target - direct, cloudinit")
 
+	install := false
+	flag.BoolVar(&install, "install", install, "If true, will install a systemd unit instead of running directly")
```

Comment:
Is the `--install` flag too ambiguous here? At a glance I didn't really understand what this was doing. Should we consider a more verbose name?